andela/travel_tool_back
# #159261779 Setup dotenv

#### What does this PR do?
This PR sets up `dotenv`

#### Description of Task to be completed?
- install dotenv package
- add a descipriton of how to use env variables in README.md
- create `.env` and `.env.example`
- use dotenv in index.js

#### How should this be manually tested?
- Clone the repo `git clone https://github.com/andela/travel_tool_back.git`
- Change into the folder `cd travel_tool_back`
- Switch to this branch `git checkout ch-dotenv-159261779`
- Install dependencies `yarn install`
- Create a `.env` file in the root of the project based off of `.env.example`
- Change the `PORT` to your preferred port
- Start the server  `yarn start:dev`
- Check to see if the server is running on the port you specified

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#159261779](https://www.pivotaltracker.com/story/show/159261779)

#### Screenshots (if appropriate)
N/A


#### Questions:
N/A
